doc/sphinx/source/contributing/rules.md

```diff
@@ -1,6 +1,5 @@
-```eval_rst
 .. _rules:
-```
+
```

Comment:
why did you remove this?

Reply:
Was considering changing .md to .srt but then decided that is too much effort.